eox_nelp/settings/common.py

```diff
@@ -33,5 +36,10 @@ def plugin_settings(settings):
     settings.FUTUREX_API_CLIENT_ID = 'my-test-client-id'
     settings.FUTUREX_API_CLIENT_SECRET = 'my-test-client-secret'
 
-    if COURSE_CREATOR_APP not in settings.INSTALLED_APPS:
+    if hasattr(settings, "INSTALLED_APPS") and COURSE_CREATOR_APP not in settings.INSTALLED_APPS:
         settings.INSTALLED_APPS.append(COURSE_CREATOR_APP)
+
```

Comment:
probably this will be no necessary

Reply:
you are meaning the white line?
![image](https://user-images.githubusercontent.com/51926076/232618908-c27f5dee-d8a3-4be7-aa57-7f9a42ad4153.png)

Reply:
whole block

Reply:
it fails me if I do not register the task(in the worker). At the moment the notifications task file is not called or imported in a principal file. So celery couldn't start a not registered task.

Reply:
That's why I said that probably this is not necessary, for reference check this [PR](https://github.com/eduNEXT/eox-nelp/pull/45) I didn't need this